### ISSUE TITLE
Slack community link invalid

### ISSUE THREAD

**Issue by AidanDelaney** [paketo-buildpacks/community#156]
The link to https://slack.paketo.io/ in README.md is invalid

## Expected Behavior
I was expecting to be redirected to the current Slack for packeto discussion.

**Comment by dmikusa** [comment]
Yes, we've been struggling with this for a bit. I thought we had it fixed but seems to be broken for me. What's weird is that it works if you're already logged into Paketo Slack, so it's deceptive.

In the meantime, you can use this [invite link](https://join.slack.com/t/paketobuildpacks/shared_invite/zt-2aj9kwj0m-6wCMwHOCfqYJiejjIZx9EA). It should not expire.

**Comment by sgaist** [comment]
Hi,

Is this address supposed to be the same as the original `paketobuildpacks.slack.com` workspace ?

**Comment by hany** [comment]
Just to add to this, it appears that the Slack invite link on your website here: https://paketo.io/community/ is using an expired invite.
